docs/users_guide/landice/test_groups/ensemble_generator.rst

```diff
@@ -102,20 +92,37 @@ The test-case-specific config options are:
    # Additional runs can be added and run to an existing ensemble
    # without affecting existing runs, but trying to set up a run
    # that already exists may result in unexpected behavior.
-   # Run numbers should be zero-based
+   # Run numbers should be zero-based.
+   # If using uniform sampling, start_run should be 0 and end_run should be
+   # equal to max_samples, otherwise unexpected behavior may result.
```

Comment:
See comments on the cfg file itself, which should also be addressed here.

Reply:
Addressed in debda1985f81d3714c95386aa66145013b39d682